horace_core/sqw/PixelData/@PixelData/PixelData.m

```diff
@@ -448,6 +450,7 @@ function delete(obj)
                 obj.write_dirty_page_();
```

Comment:
does this `obj.write_dirty_page_`, `if obj.page_is_dirty_()` check need the additional `dirty_page_edited` check (c.f. `advance()`)?